lib/bugspots/scanner.rb

```diff
@@ -14,8 +14,9 @@ def self.scan(repo, branch = "master", depth = 500, regex = nil)
     end
 
     walker = Rugged::Walker.new(repo)
-    walker.sorting(Rugged::SORT_TOPO | Rugged::SORT_REVERSE)
```

Comment:
I don't need the `default` depth value and remove here.

https://github.com/igrigorik/bugspots/pull/35/files#diff-f31d6cd0b5a7cc065510264e6fa8e7e3L44
https://github.com/igrigorik/bugspots/pull/35/files#diff-e33b85f28abaec29ab84b822091a6048R7

Do you mean that whether `depth` param itself should be applied or not?

My motivation was to apply shallow cloned repository in my CI environment where full git clone takes a long time.
But I already resolved this problem and now apply full cloned repository. If you think `depth` itself is unnecessary, please close this PR.